src/test/channelFilterRouteSpec.js

```diff
@@ -0,0 +1,66 @@
+var channelFilterRoutes = [
```

Comment:
You are missing a couple of routes in this list.
- domains/:domainId/:objectType/search
- framework/list